common/changes/autorest/feature-configuration-v2_2021-02-05-19-10.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "autorest",
+      "comment": "REthing config",
```

Do these messages need to be updated?

```suggestion
      "comment": "Use the new @autorest/configuration package",
```